extra-devel/cuda/autobuild/overrides/usr/lib/pkgconfig/accinj64.pc

```diff
@@ -0,0 +1,9 @@
+cudaroot=/usr/lib/cuda
+libdir=${cudaroot}/targets/x86_64-linux/lib
+includedir=${cudaroot}/targets/x86_64-linux/include
+
+Name: accinj64
+Description: OpenACC 64-bit Injection Library
+Version: 11.0
```

Comment:
Please try and generate all these pkgconfig files, so that you won't have to change them every time there is a major version update.